Library/test-helpers/lib.sh

```diff
@@ -612,6 +612,38 @@ limeStopAgent() {
 true <<'=cut'
 =pod
 
+=head2 limeKeylimeTenant
+
+Run the keylime tenant on localhost or in container with specified tenant commands.
+
+    limeKeylimeTenant TENANT_CMD
+
+=over
+
+=item
+
+    TENANT_CMD - Set of commands which tenant run.
+
+=back
+
+Returns 0 when the stop was successful, non-zero otherwise.
+
+=cut
+
+limeKeylimeTenant() {
+
+    local TENANT_CMD=$1
```

Comment:
I think we could avoid using extra `"` around the cmd argument while calling `limeKeylimeTenant` by using `$@`, i.e. either pass `"$@" ` to the containerized tenant or `$@` to keylime_tenant.